frontend/doc-site/04__Analyze Public Data/4_4__Annotate.mdx

```diff
@@ -0,0 +1,182 @@
+# Automated Annotation of Single Cell Data (EXPERIMENTAL)
+
+CZ CELLxGENE Annotate enables label transfer and reference mapping from expertly curated reference atlases to your newly generated and QC'd single cell datasets. Your data can be annotated using CELLxGENE Desktop along with any of our available models. CELLxGENE annotation augments your dataset with the following new information:
+- transferred cell type labels (a.k.a. the most probable cell type)
+- uncertainty scores (measure of the confidence of the predicted label - takes on a value between 0 and 1 [0 = confident, 1 = uncertain])
+- a projection of the query dataset into the reference embedding, and a newly computed UMAP based on that projection
+
+After you successfully complete the automatic annotation workflow, you can explore and refine your predictions using [standard features in CELLxGENE](https://cellxgene.cziscience.com/docs/04__Analyze%20Public%20Data/4_1__Hosted%20Tutorials) such as:
+- visualizing clusters by predicted cell type
+- plotting marker genes
+- performing differential expression
+- refining cell type labels by creating custom annotations
+
+**Disclaimer:** The Annotation feature in CELLxGENE is new and experimental. We intend for the results generated by the workflow to be a starting point for annotating your single cell dataset. If you run into issues completing this workflow, we would love to hear about your experience. Feel free to reach out to [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) to get help running the workflow or to forward constructive feedback. In the meantime, in order to obtain the best results, we make the following recommendations:
+- use this predictive annotation workflow with data generated by 10X platforms
+- use the tissue model which is most closely related to the tissue profiled in your single cell experiment (currently available models are: lung, pan-tissue immune)
+
+## Installation of CELLxGENE[annotate]
+
+
+You can install the latest experimental release of CELLxGENE to gain access to the automatic annotation subcommand. It is highly recommended to perform installation in a clean virtual environment (such as conda or venv) with python version 3.9+. Find examples of virtual environment setup using venv and conda below
+
+### venv setup
+
+For an overview of `venv`, refer [here](https://docs.python.org/3/tutorial/venv.html), for installation guidance refer [here](https://github.com/pyenv/pyenv#installation)
```

Comment:
I think we can remove the "installation guidance" part, since `venv` is a module that is installed along with Python (i.e. nothing more to install).  `pyenv`, confusingly, is a tool to manage multiple Python versions.

Reply:
oh wow, didn't realize that - that's not great guidance - agreed on the change!

Reply:
I removed the 'installation guidance' part and replaced the text with this:

'For an overview of venv, refer to the link in the above section. If you are on an apple silicon machine (i.e mac with M1/M2 processor), you must use conda as an environment manager (refer to the section directly below).'